### PR TITLE
Accessability checks

### PR DESCRIPTION
**Pull request by AhmedHossam01**
Pages titles - change to Ahmed Hossam's Portfolio.
Image text alternatives ("alt text") - Make alt text more clear.
Text headings - Changed h4 to h2 to improve document hierarchy.
Color contrast - No Issues.
Resize - No Issues.
Interaction - Add labels to the contact form.
Moving content - No issues.
Multimedia - No issues.
Basic Structure - No issues.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>My Portfolio</title>
+  <title>Ahmed Hossam's Portfolio - Homepage</title>
```
Comment (AhmedHossam01) [inline index.html:17]: Pages titles - change to Ahmed Hossam's Portfolio.

File: index.html [hunk 2]
```diff
@@ -55,22 +55,22 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
     <ul class="headline__icons">
       <li>
         <a href="#">
-          <img src="./images/linkedin.svg" alt="linkedin" />
+          <img src="./images/linkedin.svg" alt="linkedin icon" />
```
Comment (AhmedHossam01) [inline index.html:58]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 3]
```diff
@@ -55,22 +55,22 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
...
         </a>
       </li>
       <li>
         <a href="#">
-          <img src="./images/github.svg" alt="github" />
+          <img src="./images/github.svg" alt="github icon" />
```
Comment (AhmedHossam01) [inline index.html:63]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 4]
```diff
@@ -55,22 +55,22 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
...
         </a>
       </li>
       <li>
         <a href="#">
-          <img src="./images/twitter.svg" alt="twitter" />
+          <img src="./images/twitter.svg" alt="twitter icon" />
```
Comment (AhmedHossam01) [inline index.html:68]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 5]
```diff
@@ -55,22 +55,22 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
...
         </a>
       </li>
       <li>
         <a href="#">
-          <img src="./images/medium.svg" alt="medium" />
+          <img src="./images/medium.svg" alt="medium icon" />
```
Comment (AhmedHossam01) [inline index.html:73]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 6]
```diff
@@ -79,11 +79,11 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
   <section class="works">
     <div class="works__single">
       <div class="works__single__img_container">
-        <img class="works__single__img" src="images/project2.png" alt="project 1" />
+        <img class="works__single__img" src="images/project2.png" alt="Multi-Post Stories" />
```
Comment (AhmedHossam01) [inline index.html:82]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 7]
```diff
@@ -79,11 +79,11 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
...
       </div>
 
       <div class="works__single__holder">
-        <h4 class="works__single__title">Multi-Post Stories</h4>
+        <h2 class="works__single__title">Multi-Post Stories</h2>
```
Comment (AhmedHossam01) [inline index.html:86]: Text headings - Changed h4 to h2 to improve document heirarchy.

File: index.html [hunk 8]
```diff
@@ -116,11 +116,11 @@ <h4 class="works__single__title">Multi-Post Stories</h4>
 
     <div class="works__single">
       <div class="works__single__img_container">
-        <img class="works__single__img" src="images/project1.png" alt="project 1" />
+        <img class="works__single__img" src="images/project1.png" alt="Tonic" />
```
Comment (AhmedHossam01) [inline index.html:119]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 9]
```diff
@@ -116,11 +116,11 @@ <h4 class="works__single__title">Multi-Post Stories</h4>
...
       </div>
 
       <div class="works__single__holder">
-        <h4 class="works__single__title">Tonic</h4>
+        <h2 class="works__single__title">Tonic</h2>
```
Comment (AhmedHossam01) [inline index.html:123]: Text headings - Changed h4 to h2 to improve document heirarchy.

File: index.html [hunk 10]
```diff
@@ -167,22 +167,22 @@ <h2 class="headline__title headline__title--half">About Myself</h2>
       <ul class="headline__icons">
         <li>
           <a href="#">
-            <img src="./images/linkedin.svg" alt="linkedin" />
+            <img src="./images/linkedin.svg" alt="linkedin icon" />
```
Comment (AhmedHossam01) [inline index.html]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 11]
```diff
@@ -167,22 +167,22 @@ <h2 class="headline__title headline__title--half">About Myself</h2>
...
           </a>
         </li>
         <li>
           <a href="#">
-            <img src="./images/github.svg" alt="github" />
+            <img src="./images/github.svg" alt="github icon" />
```
Comment (AhmedHossam01) [inline index.html]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 12]
```diff
@@ -167,22 +167,22 @@ <h2 class="headline__title headline__title--half">About Myself</h2>
...
           </a>
         </li>
         <li>
           <a href="#">
-            <img src="./images/twitter.svg" alt="twitter" />
+            <img src="./images/twitter.svg" alt="twitter icon" />
```
Comment (AhmedHossam01) [inline index.html]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 13]
```diff
@@ -167,22 +167,22 @@ <h2 class="headline__title headline__title--half">About Myself</h2>
...
           </a>
         </li>
         <li>
           <a href="#">
-            <img src="./images/medium.svg" alt="medium" />
+            <img src="./images/medium.svg" alt="medium icon" />
```
Comment (AhmedHossam01) [inline index.html]: Image text alternatives ("alt text") - Make alt text more clear.

File: index.html [hunk 14]
```diff
@@ -250,15 +250,18 @@ <h2>Contact me</h2>
         <form action="https://formspree.io/f/mrgjezqe" method="post" class="contact-form">
           <ul>
             <li>
+              <label for="name">Your Name:</label>
```
Comment (AhmedHossam01) [inline index.html:245]: Interaction - Add labels to contact form.

File: index.html [hunk 15]
```diff
@@ -250,15 +250,18 @@ <h2>Contact me</h2>
...
               <input name="name" placeholder="Type your name..." type="text" class="input" maxlength="30" required />
             </li>
 
             <li>
+              <label for="_replyto">Your Email:</label>
               <input name="_replyto" placeholder="Type your email..." type="email" class="input" maxlength="30"
                 required />
```
Comment (AhmedHossam01) [inline index.html]: Interaction - Add labels to contact form.

File: index.html [hunk 16]
```diff
@@ -250,15 +250,18 @@ <h2>Contact me</h2>
...
             </li>
 
             <li>
+              <label for="message">Messages:</label>
```
Comment (AhmedHossam01) [inline index.html:257]: Interaction - Add labels to contact form.

File: index.html [hunk 17]
```diff
@@ -26,7 +26,7 @@
     <a href="#" class="navbar__logo">Ahmed</a>
     <nav>
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** Your site has passed so many tests, however, there are a few tests to be passed. Kindly use this [tool to check the accessibility of your website.](https://wave.webaim.org/). The tools also give suggestions on how to improve the page to pass all tests.

File: index.html [hunk 18]
```diff
@@ -55,22 +55,22 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
     <ul class="headline__icons">
       <li>
         <a href="#">
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** Most of your anchor tags are set to your root `<a href="#">`. Kindly check your anchor tags for those that would not point to another page. Let's remove them, these would greatly improve the accessibility of your website.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Team👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- descriptive PR comments✔️
- descriptive commit messages✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 17 above)
Comment (AhmedHossam01) [reply index.html]: Hi @brytebee,
Thanks for your review and I would like to ask about the contrast issues. I follow a specific design template in Figma and it has its own color decisions. Do I still have to change it and make my own decisions to improve accessibility when it comes to "contrast"?

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @AhmedHossam01** , 

### **Congratulations! 🎉 Project Approved ✅**

You did a great job implementing the requested changes :+1: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**